_docs/services/relational-database.md

```diff
@@ -15,39 +15,39 @@ If your application uses relational databases for storage, you can use the AWS R
-`small-psql-redundant`      | Dedicated redundant small RDS PostgreSQL DB instance                         | AWS RDS Latest Default |
-`medium-psql`               | Dedicated medium RDS PostgreSQL DB instance                                  | AWS RDS Latest Default |
-`medium-psql-redundant`     | Dedicated redundant medium RDS PostgreSQL DB instance                        | AWS RDS Latest Default |
-`medium-gp-psql`            | Dedicated higher workload medium RDS PostgreSQL DB instance                  | AWS RDS Latest Default |
-`medium-gp-psql-redundant`  | Dedicated higher workload redundant medium RDS PostgreSQL DB instance        | AWS RDS Latest Default |
-`large-gp-psql`             | Dedicated higher workload large RDS PostgreSQL DB instance                   | AWS RDS Latest Default |
-`large-gp-psql-redundant`   | Dedicated higher workload redundant large RDS PostgreSQL DB instance         | AWS RDS Latest Default |
-`xlarge-gp-psql`            | Dedicated higher workload x-large RDS PostgreSQL DB instance                 | AWS RDS Latest Default |
-`xlarge-gp-psql-redundant`  | Dedicated higher workload redundant xlarge RDS PostgreSQL DB instance        | AWS RDS Latest Default |
-`small-mysql`               | Dedicated small RDS MySQL DB instance                                        | AWS RDS Latest Default |
-`small-mysql-redundant`     | Dedicated redundant small RDS MySQL DB instance                              | AWS RDS Latest Default |
-`medium-mysql`              | Dedicated medium RDS MySQL DB instance                                       | AWS RDS Latest Default |
-`medium-mysql-redundant`    | Dedicated redundant medium RDS MySQL DB instance                             | AWS RDS Latest Default |
-`medium-gp-mysql`           | Dedicated higher workload medium RDS MySQL DB instance                       | AWS RDS Latest Default |
-`medium-gp-mysql-redundant` | Dedicated higher workload redundant medium RDS MySQL DB instance             | AWS RDS Latest Default |
-`large-gp-mysql`            | Dedicated higher workload large RDS MySQL DB instance                        | AWS RDS Latest Default |
-`large-gp-mysql-redundant`  | Dedicated higher workload redundant large RDS MySQL DB instance              | AWS RDS Latest Default |
-`xlarge-gp-mysql`           | Dedicated higher workload x-large RDS MySQL DB instance                      | AWS RDS Latest Default |
-`xlarge-gp-mysql-redundant` | Dedicated higher workload redundant x-large RDS MySQL DB instance            | AWS RDS Latest Default |
-`medium-oracle-se2`         | Dedicated medium RDS Oracle SE2 DB                                           | AWS RDS Latest Default |
+`micro-psql`                | Single-AZ RDS instance of PostgreSQL, minimum 1 core, minimum 1 GiB memory                                   | AWS RDS Latest Default |
```

Comment:
I wonder whether we should include this table on the website since it can easily get out of date and we instruct people on how to get this information using the CLI below

Reply:
We include it in the docs because people have a hard time imagining how things work before they actually get to grips with the platform, but yeah, there is always the risk of it drifting out of date.

The CSB takes the novel approach of having the brokerpak service catalog metadata act as the source of truth, explicitly generating HTML that can then be grabbed for use in CI/CD processes. See for example https://ssb-eks-gsa-datagov-management.app.cloud.gov/docs In future maybe we could look at doing something like that with the `cf` CLI output, just to fill in this table whenever cg-site is deployed.

Reply:
> In future maybe we could look at doing something like that with the `cf` CLI output, just to fill in this table whenever cg-site is deployed.

Or better, build it from [`cf curl /v3/service_plans`](https://v3-apidocs.cloudfoundry.org/version/3.117.0/index.html#list-service-plans) since that's already in JSON and we don't have to parse command output.